cnf.py

```diff
@@ -21,6 +21,6 @@ def load_cnf():
 
 config = {
     'host': remote,
-    'username': user,
-    'password': password
+    'username': "s53922",
```

Comment:
That's your confidential database credentials, you definitely shouldn't have uploaded these. In fact you should not be editing this file in any case.